tests/hashing/subtests/asset-cascade/webpack/webpack.config.js

```diff
@@ -10,20 +10,12 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+const path = require('path');
 module.exports = [1, 2, 3].map(src => ({
   entry: `./src${src}/index.js`,
```

Comment:
I noticed that the `main.[hash].js` hash is different in the compiled output of `src1` and `src2`, despite the directory contents being identical. That seemed surprising given that the filename template is using `[contenthash]`. When I look at the bundles, the only difference is the generated numeric module identifiers.

I think it's because they're generated using the context-relative path, which technically fails the test. However, I think we can just change the test in this case so that the `srcN` is the root directory rather than the entry path. We did the same for both Parcel and Browserify.

```suggestion
  context: path.resolve(__dirname, 'src' + src),
  entry: `./index.js`,
```